package.json

```diff
@@ -94,7 +94,7 @@
     "hardhat:ganache": "./hardhat/scripts/start-ganache.sh",
     "hardhat:test": "yarn --cwd hardhat test",
     "test": "yarn hardhat:ganache --exec craco test --runInBand",
-    "test:ci": "yarn hardhat:ganache --exec ./scripts/ci.sh",
+    "test:ci": "CI=true yarn hardhat:ganache --exec ./scripts/ci.sh",
```

Comment:
after the incident with vercel, do we want to change this to something besides CI ? @topocount

Reply:
@crabsinger @levity This would be a significant change, and I'd like to discuss this in depth about how we want to evolve environment configs mid to long term.

For starters, Vercel has environment variables (which we now use) to prevent the issue we ran into. The CI variable is meant to emulate the CI environment. That is the intent here, since ganache and cypress also can run in a local dev environment.

Rather than creating a monolithic variable, I'd rather have some canary var, that if detected in a staging or production build, will stop the build and error, like a stopship.